### ISSUE TITLE
[travis] Coverage.py warning: No data was collected. (no-data-collected)

### ISSUE THREAD

**Issue by sbellem** [initc3/HoneyBadgerMPC#89]
Seems to be a recent problem. See https://travis-ci.org/initc3/HoneyBadgerMPC/jobs/440765027#L599 for instance.

A quick possible workaround may be to add `__init__.py` file under the `tests` dir.

#### Related resources:
https://docs.pytest.org/en/latest/goodpractices.html
http://thomas-cokelaer.info/blog/2017/01/pytest-cov-collects-no-data-on-travis/
https://github.com/pytest-dev/pytest-cov/issues/176
https://github.com/pytest-dev/pytest-cov/issues/98

**Comment by amiller** [comment]
Is fixed now? @Drake-Eidukas can you close if confirm this was fixed by the pyroc commit above?

**Comment by noudald** [comment]
@amiller, I referenced #89 in a completely unrelated repository. Because I encountered the same issue as here. GitHub likes to mention these references. To be clear, the pyroc fix commit was not committed to HoneybadgerMPC!

I think the issue is with Travis-CI. However I can confirm that the `__init__.py` workaround works for the pyroc project, so it will possible work for your repository as well.

I hope this helps.
